packages/core/src/utilities/isValidVolume.ts

```diff
@@ -0,0 +1,51 @@
+import { metaData } from '..';
+import isEqual from './isEqual';
+
+function isValidVolume(imageIds: string[]): boolean {
```

Comment:
Please document exactly what is considered a valid volume here - it seems like it isn't quite the consistent spacing that is used in some other places.